src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -480,6 +505,103 @@ public override async Task RunAsync()
             await runTask;
         }
 
+        private async Task<bool> HandleRuntimeParam(bool isCurrentProcessNet6Build, bool isVerbose)
+        {
+            if (string.Equals(SetHostRuntime, "default", StringComparison.OrdinalIgnoreCase))
+            {
+                if (isCurrentProcessNet6Build)
+                {
+                    if (isVerbose)
+                    {
+                        ColoredConsole.WriteLine(VerboseColor("Selected out-of-process host"));
+                    }
+                    await StartHostAsChildProcessAsync(isOutOfProc: true);
+                    return true;
+                }
+            }
+            else if (string.Equals(SetHostRuntime, InProc8HostRuntime, StringComparison.OrdinalIgnoreCase))
+            {
+                if (isCurrentProcessNet6Build && ShouldLaunchInProcNet8AsChildProcess() && await IsInProcNet8Enabled())
+                {
+                    if (isVerbose)
+                    {
+                        ColoredConsole.WriteLine(VerboseColor($"Selected {InProc8HostRuntime} host"));
+                    }
+                    await StartHostAsChildProcessAsync(isOutOfProc: false);
+                    return true;
+                }
+            }
+            else if (string.Equals(SetHostRuntime, InProc6HostRuntime, StringComparison.OrdinalIgnoreCase))
+            {
+                if (isVerbose)
```

Comment:
Seeing this pattern used a lot, could this be it's own function? "LogVerboseMessage"